### DISCUSSION TITLE
Unable to import if installed as local package

### DISCUSSION THREAD

**Discussion by sunnyxxchen** [NatLabRockies/floris#1005] [Q&A] [answered]
I'm trying to use a local installation of FLORIS to modify some of the source code. I cloned the repo and installed with `pip install -e floris` as per the installation instructions. However, I'm now running into some kind of issue with the PATH. If I start Python and import outside of the `floris` directory (where I cloned the repo), then the package doesn't import any of the functions correctly. Depending on the directory, running `help(floris)` will produce something like this:

```Help on package floris:

NAME
    floris

PACKAGE CONTENTS


FILE
    (built-in)
```

I had previously installed using `pip install floris`, if that makes a difference. Thank you for your help!

**Comment by misi9170** [comment]
Hi @sunnyxxchen ,

This comment might help: https://github.com/NREL/floris/discussions/979#discussioncomment-10608687 . In particular, if you run `pip show floris`, does it print the path to where you cloned the repository, or does it print something else? You may need to start fresh with a new conda environment, or you could try `pip uninstall floris` before retrying the local install `pip install -e floris`.

**Reply by sunnyxxchen** [reply]
Thanks. `pip show floris` gives both a "location" and "editable project location". The location shows my Conda environment package directory, while the editable project location gives the cloned repo. If there is no easy fix, I may go ahead and retry with a new environment.

**Reply by misi9170** [reply, marked as answer]
Hmm. One other thing to try is to just run `pip list` and then see what shows up next to the `FLORIS` entry. When I do that with a locally installed version, I see something like this:
```
FLORIS                        4.1.1         /Users/mymachine/floris
```
whereas if it is pip install from PyPI, the path disappears and I see:
```
FLORIS          4.1.1
```
that is, no location is shown. 

One way or another, it seems as though conda has gotten mixed up about the location of the FLORIS package, and I think a fresh environment with floris locally installed would likely fix it.

**Reply by sunnyxxchen** [reply]
It turns out that I had originally installed FLORIS with conda instead of pip. Trying `conda remove floris` did not fix the problem, so I went ahead and started with a new environment. Thank you for your help!
